Dockerfile

```diff
@@ -29,12 +48,11 @@ RUN addgroup \
 
 ENV USER git
 ENV GITEA_CUSTOM /data/gitea
-ENV GODEBUG=netdns=go
 
 VOLUME ["/data"]
 
 ENTRYPOINT ["/usr/bin/entrypoint"]
 CMD ["/bin/s6-svscan", "/etc/s6"]
 
 COPY docker /
-COPY gitea /app/gitea/gitea
+COPY --from=build-env /go/src/code.gitea.io/gitea/gitea /app/gitea/gitea
```

Comment:
This way gitea binary will become writable for user git. It is out of scope for this PR - I write it for future reference. Not a blocker.

Reply:
Are you sure ? I haven't tested but that should be root the owner and permissions should be the same as go build result.

Reply:
I'm sure. It's because of [this](https://github.com/go-gitea/gitea/blob/master/docker/etc/s6/gitea/setup#L40).

Reply:
Ok that should be fix but it was already the case before and not introduce by this PR.

Reply:
Yes, this is exactly why I wrote:

> This is out of scope for this PR - I'm just writing it for future reference. Not a blocker.

;)

Reply:
Sorry misread, and thinking that specific line introduce that XD.

Reply:
No problem. ;)